serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -15,20 +15,20 @@ ARG cuda_version=cu124
 ARG python_version=3.10
 ARG TORCH_VERSION=2.3.1
 ARG djl_version=0.29.0~SNAPSHOT
-ARG transformers_version=4.40.0
-ARG accelerate_version=0.29.3
-ARG tensorrtlibs_version=10.0.1
+ARG transformers_version=4.42.4
+ARG accelerate_version=0.32.1
+ARG tensorrtlibs_version=10.1.0
 # %2B is the url escape for the '+' character
-ARG trtllm_toolkit_version=0.10.0%2Bnightly
-ARG trtllm_version=v0.10.0
+ARG trtllm_toolkit_version=0.11.0%2Bnightly
+ARG trtllm_version=v0.11.0
 ARG cuda_python_version=12.4
 ARG peft_version=0.10.0
 ARG triton_version=r24.04
 ARG trtllm_toolkit_wheel="https://publish.djl.ai/tensorrt-llm/toolkit/tensorrt_llm_toolkit-${trtllm_toolkit_version}-py3-none-any.whl"
-ARG trtllm_wheel="https://djl-ai.s3.amazonaws.com/publish/tensorrt-llm/${trtllm_version}/tensorrt_llm-0.10.0-cp310-cp310-linux_x86_64.whl"
+ARG trtllm_wheel="https://publish.djl.ai/tensorrt-llm/${trtllm_version}/tensorrt_llm-0.11.0-cp310-cp310-linux_x86_64.whl"
```

Comment:
@ydm-amazon this will allow us to use cloudfront (an AWS services) that do global CDN on the artifacts we have and provide 10x speed up on downloading. Using raw HTTP url for S3 will be slower than that.